src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -887,26 +887,79 @@ function chip_tests_item_has_list(options)
   });
 }
 
+async function chip_tests_only_clusters(options)
+{
+  const clusters         = await getClusters(this);
+  const testOnlyClusters = clusters.filter(cluster => isTestOnlyCluster(cluster.name));
+  return asBlocks.call(this, Promise.resolve(testOnlyClusters), options);
+}
+
+async function chip_tests_only_cluster_commands(options)
+{
+  const commands = await getCommands(this, this.name);
+  return asBlocks.call(this, Promise.resolve(commands), options);
+}
+
+async function chip_tests_only_cluster_command_parameters(options)
+{
+  return asBlocks.call(this, Promise.resolve(this.arguments), options);
+}
+
+async function chip_tests_only_cluster_responses(options)
```

Comment:
I don't see where the "test only" part is done.  Is this just because we assume this is used inside `chip_tests_only_clusters`?  Should document that, if so.